packages/components/primary-action-dropdown/src/primary-action-dropdown.tsx

```diff
@@ -125,12 +132,15 @@ const DropdownChevron = forwardRef<HTMLButtonElement, TDropdownChevron>(
       onClick={props.onClick}
       isDisabled={props.isDisabled}
       css={[
-        ...getButtonStyles(props.isDisabled),
+        ...getButtonStyles(props.isDisabled, props.isNewTheme),
         css`
           padding: 0 ${designTokens.spacing10};
           border-left: 1px solid ${designTokens.colorNeutral};
           border-radius: 0 ${designTokens.borderRadius6}
             ${designTokens.borderRadius6} 0;
+          border: ${props.isNewTheme &&
+          `1px solid ${designTokens.colorNeutral}`};
+          margin-left: ${props.isNewTheme && '-1px'};
```

Comment:
**nit** An alternative would be maybe to use a new design token here.
Example:
```
// default theme
border-for-primary-action-dropdown-chevron: 'none'
margin-left-for-primary-action-dropdown-chevron: '0'

// new theme
border-for-primary-action-dropdown-chevron: '1px solid var(--color-neutral)'
margin-left-for-primary-action-dropdown-chevron: '-1px'
```

Reply:
@CarlosCortizasCT Thank you for this! 🙏 I tried to write even a bit more cleaner these few lines, you can check them here, and let me know what you think - https://github.com/commercetools/ui-kit/pull/2450/commits/3800a9d7995a91b2745c15b667c7ff8ecc04eed1